media/videostitcher/create_vod_session.go

```diff
@@ -21,7 +21,7 @@ import (
 	"io"
 
 	stitcher "cloud.google.com/go/video/stitcher/apiv1"
-	"cloud.google.com/go/video/stitcher/apiv1/stitcherpb"
+	stitcherstreampb "cloud.google.com/go/video/stitcher/apiv1/stitcherpb"
```

Comment:
Just curious, is there a reason for the new alias?

Reply:
Changed it to match the other samples.